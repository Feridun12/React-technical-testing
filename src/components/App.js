import "../styles/App.css";
// components
import Search from "./Search";

function App() {
  return (
    <div className="App">
      <img className= "nasa-picture" 
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg" />
      <Search />
    </div>
  );
}

export default App;
