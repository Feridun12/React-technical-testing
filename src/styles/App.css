.App {
display: flex;
align-items: center;
justify-content: center;
flex-direction: column;
}

.nasa-picture {
width: 70px;
height: 70px;
}