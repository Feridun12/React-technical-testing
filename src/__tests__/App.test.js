import { render } from '@testing-library/react';
import App from '../components/App';

test('App renders correctly:snapshot testing', () => {
  const {asFragment} = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});