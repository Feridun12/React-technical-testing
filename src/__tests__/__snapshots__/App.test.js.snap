// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly:snapshot testing 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      class="nasa-picture"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <input
      class="search-field"
      type="text"
    />
  </div>
</DocumentFragment>
`;
